tests/lava/proc/monitor/test_monitors.py

```diff
@@ -0,0 +1,288 @@
+        probe_data = data[some_proc.name][some_proc.s.name]
+
+        # Check if the collected data match the expected data
+        self.assertTrue(np.all(probe_data == np.array([1, 2, 3, 4, 5, 6])))
+
+        # stop running
+        some_proc.stop()
+
+    def test_monitor_collects_voltage_and_spike_data_from_lif_neuron(self):
+        """Check if two different Monitor process can monitor voltage (Var) and
+        s_out (OutPort) of a LIF neuron. Check the collected data with
+        expected data.
+        Note: The LIF neuron integrate the given bias, voltage accumulates
+        and once pass the threshold, there will be a spike outputted and
+        voltage will be reset to zero
+        """
+        # Setup
+        monitor1 = Monitor()
+        monitor2 = Monitor()
+        shape = (1,)
+        num_steps = 6
+        neuron = LIF(shape=shape,
+                     vth=3,
+                     bias=1)
+
+        rcnd = RunSteps(num_steps=num_steps)
+        rcfg = LifRunConfig(select_tag='floating_pt')
+
+        # Probe voltage of LIF with the first monitor
+        monitor1.probe(target=neuron.v, num_steps=num_steps)
+
+        # Probe spike output of LIF with the second monitor
+        monitor2.probe(target=neuron.s_out, num_steps=num_steps)
+
+        # Run
+        neuron.run(condition=rcnd, run_cfg=rcfg)
+
+        # Get data from both monitor
+        data1 = monitor1.get_data()
+        data2 = monitor2.get_data()
+
+        # stop
+        neuron.stop()
+
+        # Access the relevant data in the corresponding data dicts
+        volt_data = data1[neuron.name][neuron.v.name]
+        spike_data = data2[neuron.name][neuron.s_out.name]
+
+        # Check if this data match the expected data
+        print(volt_data)
+        print(spike_data)
+        self.assertTrue(np.all(volt_data == np.array([1, 2, 0, 1, 2, 0])))
+        self.assertTrue(np.all(spike_data == np.array([0, 0, 0, 1, 0, 0])))
+
+    def test_monitor_collects_voltage_and_spike_data_from_population_lif(self):
+        """Check if two different Monitor process can monitor voltage (Var) and
+        s_out (OutPort) of a population of two LIF neurons. Check the
+        collected data with expected data.
+        Note: The LIF neurons integrate the given bias, voltage accumulates
+        and once pass the threshold, there will be a spike outputted and
+        voltage will be reset to zero
+        """
+        # Setup
+        monitor1 = Monitor()
+        monitor2 = Monitor()
+        shape = (2,)
+        num_steps = 6
+        neuron = LIF(shape=shape,
+                     vth=3,
+                     bias=1)
+
+        # Probe voltage of LIF neurons with the first monitor
+        monitor1.probe(target=neuron.v, num_steps=num_steps)
+
+        # Probe spike output of LIF neurons with the second monitor
+        monitor2.probe(target=neuron.s_out, num_steps=num_steps)
+
+        # Run
```

Comment:
These very short comments are not helpful - the name of the method already says it.